app/src/main/java/org/dieschnittstelle/mobile/android/skeleton/MainActivity.java

```diff
@@ -1,34 +1,93 @@
 package org.dieschnittstelle.mobile.android.skeleton;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.databinding.DataBindingUtil;
+import androidx.lifecycle.ViewModelProvider;
 
 import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.EditText;
 import android.widget.TextView;
 
 import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.google.android.material.snackbar.Snackbar;
+
+import org.dieschnittstelle.mobile.android.skeleton.databinding.ActivityMainBinding;
+import org.dieschnittstelle.mobile.android.skeleton.databinding.ActivityTaskDetailViewBinding;
+import org.dieschnittstelle.mobile.android.skeleton.model.IUserDatabaseOperation;
+import org.dieschnittstelle.mobile.android.skeleton.model.User;
+import org.dieschnittstelle.mobile.android.skeleton.viewmodel.MainViewModel;
+import org.dieschnittstelle.mobile.android.skeleton.viewmodel.TaskListViewModel;
 
 public class MainActivity extends AppCompatActivity {
 
     private TextView welcomeText;
-    private FloatingActionButton showTaskListAction;
+    private EditText emailAdress;
+    private EditText password;
+    private Button showTaskListAction;
+    private MainViewModel viewModel;
+    private IUserDatabaseOperation userDBOperation;
+    private User user = new User("", "");
```

Comment:
das soll man nicht hier initialisieren. sondern während `onCreate()` wie bei Task-Objekt in `TaskDetailViewActivity` Klasse